### PR TITLE
Luminous testing 2

### PR DESCRIPTION
**Pull request by leseb**

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test this please

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [tox]
-envlist = {jewel,kraken,rhcs}-{ansible2.2}-{xenial_cluster,journal_collocation,centos7_cluster,dmcrypt_journal,dmcrypt_journal_collocation,docker_cluster,purge_cluster,purge_dmcrypt,docker_dedicated_journal,docker_dmcrypt_journal_collocation,update_dmcrypt,update_cluster,cluster,purge_docker_cluster,update_docker_cluster,bluestore_journal_collocation,bluestore_cluster}
+envlist = {jewel,kraken,rhcs}-{ansible2.2}-{xenial_cluster,journal_collocation,centos7_cluster,dmcrypt_journal,dmcrypt_journal_collocation,docker_cluster,purge_cluster,purge_dmcrypt,docker_dedicated_journal,docker_dmcrypt_journal_collocation,update_dmcrypt,update_cluster,cluster,purge_docker_cluster,update_docker_cluster}
```
Comment (andrewschoen) [inline tox.ini]: You should add ``luminous`` in this list so it inherits all the common scenarios.  Like: ``{jewel,kraken,rhcs,luminous}-``

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+  {luminous}-{ansible2.2}-{xenial_cluster,journal_collocation,centos7_cluster,dmcrypt_journal,dmcrypt_journal_collocation,docker_cluster,purge_cluster,purge_dmcrypt,docker_dedicated_journal,docker_dmcrypt_journal_collocation,update_dmcrypt,update_cluster,cluster,purge_docker_cluster,update_docker_cluster,bluestore_journal_collocation,bluestore_cluster,bluestore_dmcrypt_journal,bluestore_dmcrypt_journal_collocation}
```
Comment (andrewschoen) [inline tox.ini]: If you add ``luminious`` to the above line you can *only* include the bluestore_* scenarios here to reduce some of the duplication.

File: tests/functional/tests/osd/test_journal_collocation.py [hunk 3]
```diff
@@ -4,4 +4,4 @@ class TestOSD(object):
     def test_osds_are_all_collocated(self, node, Command):
         # TODO: figure out way to paramaterize node['vars']['devices'] for this test
         for device in node["vars"]["devices"]:
-            assert Command.check_output("sudo blkid -s PARTLABEL -o value %s2" % device) == "ceph journal"
+            assert Command.check_output("sudo blkid -s PARTLABEL -o value %s2" % device) == ("ceph journal" or "ceph block")
```
Comment (andrewschoen) [inline tests/functional/tests/osd/test_journal_collocation.py]: ``assert Command.check_output("sudo blkid -s PARTLABEL -o value %s2" % device) in ["ceph journal", "ceph block"]`` would be better I think

Comment (leseb) [inline tests/functional/tests/osd/test_journal_collocation.py]: Thanks!

File: tox.ini (hunk 1 above)
Comment (leseb) [inline tox.ini]: Thanks!

**Pull request closed without merging**